docs/topics/azure-stack.md

```diff
@@ -0,0 +1,154 @@
+
+| Name                            | Required | Description|
+| ------------------------------- | -------- | ---------- |
+| vmsize                          | yes      | Specifies a valid [Azure Stack VM size](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-vm-sizes). |
+| distro                          | yes      | Specifies the masters' Linux distribution. Currently supported values are: `"ubuntu"` and `"aks-ubuntu-16.04"` (previously `"aks"`). `"aks-ubuntu-16.04"` is a custom image based on ubuntu-16.04 that comes with pre-installed software necessary for Kubernetes deployments. `"aks-ubuntu-16.04"` should be selected if you target a disconnected Azure Stack instance. |
+
+### agentPoolProfiles
+
+`agentPoolProfiles` are used to create agents with different capabilities.
+
+| Name                            | Required | Description|
+| ------------------------------- | -------- | ---------- |
+| vmsize                          | yes      | Describes a valid [Azure Stack VM size](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-vm-sizes). |
+| distro                          | yes      | Specifies the masters' Linux distribution. Currently supported values are: `"ubuntu"` and `"aks-ubuntu-16.04"` (previously `"aks"`). `"aks-ubuntu-16.04"` is a custom image based on ubuntu-16.04 that comes with pre-installed software necessary for Kubernetes deployments. `"aks-ubuntu-16.04"` should be selected if you target a disconnected Azure Stack instance. |
+| availabilityProfile             | yes      | Only `"AvailabilitySet"` is currently supported. |
+| acceleratedNetworkingEnabled    | yes      | Use `Azure Accelerated Networking` feature for Linux agents. This property should be always set to `"false"`. |
+
+## Azure Stack Instances Registered with Azure's China cloud
+
+If your Azure Stack instance is located in China, then the `dependenciesLocation` property of your cluster definition should be set to `"china"`. This switch ensures that the provisioning process fetches software dependencies from reachable hosts within China's mainland.
+
+## Disconnected Azure Stack Instances
+
+By default, the AKS Engine provisioning process relies on an internet connection to download the software dependencies required to create or upgrade a cluster (Kubernetes images, etcd binaries, network plugins and so on).
+
+If your Azure Stack instance is air-gapped or if network connectivity in your geographical location is not reliable, then the default approach will not work, take a long time or timeout due to transient networking issues.
+
+With these challenges in mind, you can choose to set the `distro` property of your cluster definition to `"aks-ubuntu-16.04"`. This change will instruct AKS Engine to deploy VM nodes using a base OS image call `AKS Base Image`. This custom image already contains the required software dependencies in its file system. Hence, internet connectivity won’t be required during the provisioning process.
```

Comment:
this is true in master but the rename from `aks-16.04` to `aks-ubuntu-16.04` isn't part of 0.36.2.